src/theme/components/header/headerSearchForm.ts

```diff
@@ -26,16 +21,17 @@ const HeaderSearchForm = {
       maxWidth: { md: "1312px" },
       whiteSpace: "initial",
       "> div": {
+        margin: "20px 15px 0",
         marginLeft: { md: "m", lg: "140px" },
         marginRight: { md: "m" },
       },
     },
-    fieldset: {
-      margin: { base: "20px 15px 0", md: "0" },
+    "#radio-group-search-type": {
+      margin: { base: "0 20px 20px", md: "0" },
     },
     textInput: {
       label: {
-        fontSize: { md: "18px" },
+        fontSize: "22px",
```

Comment:
In the design on the Final V1 page in Figma, the font size for the label is 28px (`heading.secondary`) for desktop and 22px (`heading.tertiary`) for mobile.

https://www.figma.com/file/FH7n0slL59s8Jp0Lmlabv8/Global-Header%3A-3rd-Radio-Button?node-id=1158%3A1177

Reply:
Also, I am not sure if there is a style somewhere setting the font size for the radio button, but the label text on the radio buttons is too large.  It is currently 18px (`heading.callout`), but it should be 14px (`text.caption`).

Reply:
I'll make the updates but just a reminder that we were suppose to build the header to look like the production header and not involve too many "new" updates. With this third radio button feature, more style changes are creeping in. This PR probably won't get merged in for a while but just wanted to point this out.

Reply:
OK, I see.  That does get a bit tricky.  However, if UI changes are being introduced into the production header, don’t those same UI changes need to be reflected here as well?